.circleci/config.yml

```diff
@@ -42,24 +42,24 @@ jobs:
               pip install -e .
               cd ..
 
-        - run:
-            name: Spin up Xvfb
-            command: |
-              /sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -screen 0 1400x900x24 -ac +extension GLX +render -noreset;
+        # - run:
+        #     name: Spin up Xvfb
+        #     command: |
+        #       /sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -screen 0 1400x900x24 -ac +extension GLX +render -noreset;
 
         # https://github.com/ContinuumIO/anaconda-issues/issues/9190#issuecomment-386508136
         # https://github.com/golemfactory/golem/issues/1019
-        - run:
-            name: Install PyQt5 dependencies
-            command: |
-              sudo apt-get install libxkbcommon-x11-0
-
-        - run:
-            name: Install 3D dependencies
-            command: |
-              pip install --user --upgrade --progress-bar off vtk
-              pip install --user --upgrade --progress-bar off mayavi
-              pip install --user --upgrade --progress-bar off PySurfer[save_movie]
+        # - run:
+        #     name: Install PyQt5 dependencies
+        #     command: |
+        #       sudo apt-get install libxkbcommon-x11-0
+
+        # - run:
+        #     name: Install 3D dependencies
+        #     command: |
+        #       pip install --user --upgrade --progress-bar off vtk
+        #       pip install --user --upgrade --progress-bar off mayavi
+        #       pip install --user --upgrade --progress-bar off PySurfer[save_movie]
```

Comment:
please keep this 3D config for a next PR. It's unrelated to the change here.

Reply:
I've opened #129